### DISCUSSION TITLE
JS shows error

### DISCUSSION THREAD

**Discussion by misko2k** [azuwis/zigbee2mqtt-networkmap#30] [Q&A]
Hi , 

Is this zigbee2mqtt-networkmap depricated ? If not , has any idea how to fix this ?

thx

![zigbeemap](https://user-images.githubusercontent.com/21138890/107857633-c6a2ac00-6e2f-11eb-9842-5d037f487796.jpg)

**Comment by azuwis** [comment]
It works here with HomeAssistant 2021.2.3 and Zigbee2mqtt 1.17.1

Please check your configuration. If it's correct, please fill the issue template with more information.
